sanic/blueprints.py

```diff
@@ -93,10 +95,16 @@ def route(self, uri, methods=frozenset({'GET'}), host=None,
         :param uri: endpoint at which the route will be accessible.
         :param methods: list of acceptable HTTP methods.
         """
+
```

Comment:
I'm fine with adding this log mesage, but do the whitespace changes here conform to the rest of the project?